src/utils.ts

```diff
@@ -390,6 +391,19 @@ export const oauthToken = async (
     worker
   );
 
+  const missingScope = getMissingScope(scope, result.scope);
+  if (missingScope.length) {
+    console.warn(
+      `The requested scopes (${scope}) are different from the scopes part of the retrieved token (${result.scope}). This could result in unexpected behavior and might eventually become unsupported. It is advised to resolve this by either:
```

Comment:
>  and might eventually become unsupported

I'm not sure we should include this; we're not sure yet what we'll do when the scopes differ but how do we communicate at this point what "unsupported" means? Also how would we explain the unexpected behaviour?

I think it would be better to just say something like:

```suggestion
      `The requested scopes (${scope}) are different from the scopes of the retrieved token (${result.scope}). This could mean that your access token may not include all the scopes that you expect. It is advised to resolve this by either:
```

WDYT?

Reply:
Oh, I thought we mentioned that we wanted to leave some kind of message that the support for this could be dropped. But sure, I will take your suggestion!